src/fault/fault_config.cpp

```diff
@@ -0,0 +1,13 @@
+#include <yaclib/fault/fault_config.hpp>
+
+void SetFrequency(uint32_t freq) {
+#ifdef YACLIB_FAULT
+  yaclib::detail::Yielder::SetFrequency(freq);
+#endif
+}
+
+void SetSleepTime(uint32_t ns) {
+#ifdef YACLIB_FAULT
+  yaclib::detail::Yielder::SetSleepTime(ns);
+#endif
+}
```

Comment:
Maybe better make it macro?

Reply:
It seems like I can't simply use ifdef inside macro definition?

Reply:
Yep